_docs/_latest/concepts/models.md

```diff
@@ -1,84 +1,80 @@
 ---
 title: Models
-order: 100
+order: 003
 ---
 
 # {{ page.title }}
 
 Starting with version 2.0, Forseti introduces the use of data models.
 
 The data model is an additional pool of data that is relational in nature,
-and is created from the flat json data in inventory, that's captured from GCP
-API responses. With the relational data, we can more easily see the
-relationships and the inheritances between resources, as well as being able
-to query for them.  One example is to see all the accesses that a service
-account may have, across all the projects.
+and is created from the flat json data in inventory. With the relational data, Forseti 
+can more easily understand the entire relationship, including inheritance between resources. Models
+allow for easier querying against the entire computed policy. For example, a data model for
+service accounts allows Forseti to easily compute what access it has across all the projects in the 
+Organization.
 
 Another important concept to keep in mind is that the data models are not meant
 to be persistent. The data models need to be created before use, and then
-destroyed after use. So, before you want to use Explain, you will need to
-create your own set of data model, and then delete them after finish using
-Explain.
+destroyed after use. So, before using Explain, you will need to create a data model set, and then
```

Comment:
I think a data model should consist of the set of tables so the user will be creating one data model and query the model using explain, each individual table doesn't qualify as a "data model".

Reply:
done